sw/airborne/modules/guidance/gvf/gvf.h

```diff
@@ -32,6 +32,13 @@
 
 #include "std.h"
 
+/** \typedef gvf_conf
```

Comment:
cool, thanks for the link. I will do it as you suggest then, probably on Sunday or Monday.

By the way, I have written the units where it was possible. Here in particular, the error signal does not have any units that make sense. In fact, they are just level sets https://en.wikipedia.org/wiki/Level_set .

The direction also does not have any preference, it is just 1 or -1 and depending on how the trajectory is defined and what are the parameters, could be left or right. I will also add a miniexplanation in the comment about this.